packages/edit-site/src/components/layout/style.scss

```diff
@@ -158,17 +127,36 @@
 	height: 100%;
 }
 
+/* stylelint-disable -- Disable reason: View Transitions not supported properly by stylelint. */
+html:active-view-transition-type(canvas-mode-edit) {
+	&::view-transition-group(toggle) {
+		animation-delay: 250ms;
```

Comment:
It's tiny, but `225ms` gives the frame enough time to move over before scaling back the site logo.